src/components/virtualRepeat/virtual-repeater.js

```diff
@@ -279,6 +313,15 @@ VirtualRepeatContainerController.prototype.handleScroll_ = function() {
   this.offsetter.style.webkitTransform = transform;
   this.offsetter.style.transform = transform;
 
+  if(this.bindTopIndex) {
+    var topIndex = Math.floor(offset / itemSize);
+    if(topIndex !== this.topIndex && topIndex < this.repeater.itemsLength) {
+      this.topIndex = topIndex;
+      this.bindTopIndex.assign(this.$scope, topIndex);
+      if (!this.$scope.$root.$$phase) this.$scope.$digest();
```

Comment:
I'm just being consistent with the rest of the file, which runs a check before each digest. I'm not an expert on Angular's digest cycle but figure it is better to be safe than have issues pop up in the future.

Feedback?